letsencrypt/client/tests/client_test.py

```diff
@@ -0,0 +1,88 @@
+"""letsencrypt.client.client.py tests."""
+import unittest
+
+import mock
+import zope.component
+
+
+class RollbackTest(unittest.TestCase):
+    """Test the rollback function."""
+    def setUp(self):
+        self.m_install = mock.MagicMock()
+        self.m_input = mock.MagicMock()
+        zope.component.getUtility = self.m_input
```

Comment:
this looks dangerous, as `zope.component.getUtility` never gets restored, patching should be used